composer.json

```diff
@@ -14,15 +14,11 @@
         "nuvoleweb/robo-config": "^3.1"
     },
     "require-dev": {
-        "openeuropa/code-review-library": "1.x-dev",
-        "phpunit/phpunit": "^9.4",
+        "openeuropa/code-review-library": "^1.0.0-alpha1",
+        "phpunit/phpunit": "^9.5",
         "slevomat/coding-standard": "^7.0 || ^8.0"
```

Comment:
Do we still need this package? I didn't find anywhere in composer.lock other packages that used this.